base/applications/rapps/unattended.cpp

```diff
@@ -312,6 +313,48 @@ HandleInfoCommand(CAppDB *db, LPWSTR szCommand, int argcLeft, LPWSTR *argvLeft)
     return TRUE;
 }
 
+static unsigned int CALLBACK
+RunGuiThread(void *nCmdShow)
+{
+    WCHAR fakecmdline[] = L"://";
```

Comment:
Why would we need a fake cmdline here?

Reply:
Because ParseCmdAndExecute needs something to chew on. It could be an empty string in theory but this is fine.

Reply:
Actually I would argue that just using an empty string would be less problematic for developers who read the code to understand what's going on.
As you can see, with this non-empty string you specified, I had to ask myself (and then you as the author), why using this string, and why not some other string, and also whether that particular choice of string is something very special the ParseCmdAndExecute function would understand...